docs/resources/msp_managed_tenant_user_api_token.md

```diff
@@ -0,0 +1,25 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cdo_msp_managed_tenant_user_api_token Resource - cdo"
+subcategory: ""
+description: |-
+  Provides a resource to generate an API token for a user in an MSP-managed tenant.
+---
+
+# cdo_msp_managed_tenant_user_api_token (Resource)
+
+Provides a resource to generate an API token for a user in an MSP-managed tenant.
```

Comment:
I'm not familiar with TF doc style, but instead of "generate an API token" should we say "manage an API token" since we are using this to manage it through the full life cycle of the token.

Reply:
That's a great point, thank you!